remoteappmanager/handlers/home_handler.py

```diff
@@ -254,7 +252,6 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         Container
         """
 
-        user_name = orm_user.name
```

Comment:
what the orm_user was is now an opaque type that we do not implement an interface for, so there is no guarantee and no public api for how this opaque type should behave, i.e. we can't make sure the object has an attribute `name`.  Afterall this is a private, "helper method that works with low level data".  Is the homogeneity of types of arguments to this single method worth a ABCUser?  I don't think so.